### PR TITLE
Implemented multiple hostAliases entries

### PR DESCRIPTION
**Pull request by IliaRN**
# Description
Added the ability to add multiple hostAliases entries.
Also added corresponding changes to csm-docs.
https://github.com/dell/csm-docs/pull/192
# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/243 |

# How Has This Been Tested?

Manually in real env

### CODE REVIEW

File: helm/csm-replication/values.yaml [hunk 1]
```diff
@@ -29,6 +29,9 @@ retryIntervalMax: 5m
 
 # HostAliases: Optional features that allows <hostname:IP> entry injection into pod's /etc/hosts file
 hostAliases:
```
Comment (maxan98) [inline helm/csm-replication/values.yaml:31]: This should be pre-commented in sample values file

File: helm/csm-replication/templates/controller.yaml [hunk 2]
```diff
@@ -279,4 +275,4 @@ spec:
       - configMap:
           name: dell-replication-controller-config
           optional: true
-        name: configmap-volume
+        name: configmap-volume
```
Comment (AronAtDell) [inline helm/csm-replication/templates/controller.yaml:278]: No newline eof.

File: helm/csm-replication/values.yaml [hunk 3]
```diff
@@ -29,6 +29,9 @@ retryIntervalMax: 5m
...
-  enableHostAliases: false
-  hostName: "foo.bar"
-  ip: "10.10.10.10"
\ No newline at end of file
+  - ip: "10.10.10.10"
+    hostnames:
+      - "foo.bar"
+  - ip: "10.10.10.11"
+    hostnames:
+      - "foo.baz"
```
Comment (AronAtDell) [inline helm/csm-replication/values.yaml:37]: No newline eof.

**Review by AronAtDell — DISMISSED** [review]
Apart from my comments, everything else looks good. Thanks!

**Review by maxan98 — APPROVED** [review]

**Review by AronAtDell — APPROVED** [review]

**Review by walker2 — CHANGES_REQUESTED** [review]
We should make corresponding changes to documentation to not forget about that later, please make the changes and link to them in this PR

**Review by walker2 — APPROVED** [review]

**Pull request merged**